src/coreComponents/physicsSolvers/wavePropagation/AcousticVTIFields.hpp

```diff
@@ -33,29 +33,53 @@ namespace fields
 namespace acousticvtifields
 {
 
```

Comment:
Please do not duplicate fields which are already inside the AcousticFields.hpp

This file should only contains fields specific for VTI

Reply:
Done thanks!